profilingdemo/src/main/java/org/kaaproject/kaa/examples/profiling/ProfilingDemoBuilder.java

```diff
@@ -0,0 +1,109 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.kaaproject.kaa.examples.profiling;
+
+import org.kaaproject.kaa.common.dto.ApplicationDto;
+import org.kaaproject.kaa.common.dto.ConfigurationDto;
+import org.kaaproject.kaa.common.dto.ConfigurationSchemaDto;
+import org.kaaproject.kaa.common.dto.EndpointGroupDto;
+import org.kaaproject.kaa.common.dto.EndpointProfileSchemaDto;
+import org.kaaproject.kaa.common.dto.UpdateStatus;
+import org.kaaproject.kaa.common.dto.ctl.CTLSchemaDto;
+import org.kaaproject.kaa.examples.common.AbstractDemoBuilder;
+import org.kaaproject.kaa.examples.common.KaaDemoBuilder;
+import org.kaaproject.kaa.server.common.admin.AdminClient;
+import org.kaaproject.kaa.server.common.utils.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * @author Maksym Liashenko
+ */
+@KaaDemoBuilder
+public class ProfilingDemoBuilder extends AbstractDemoBuilder {
+
+    private static final Logger logger = LoggerFactory.getLogger(ProfilingDemoBuilder.class);
+
+    public ProfilingDemoBuilder() {
+        super("demo/profiling");
+    }
+
+    @Override
+    protected void buildDemoApplicationImpl(AdminClient client) throws Exception {
+
+        logger.info("Loading 'Endpoint profiling demo application' data...");
+
+        loginTenantAdmin(client);
+
+        ApplicationDto profilingApplication = new ApplicationDto();
+        profilingApplication.setName("Endpoint profiling demo");
+        profilingApplication.setCredentialsServiceName("Trustful");
+        profilingApplication = client.editApplication(profilingApplication);
+
+
+        sdkProfileDto.setApplicationId(profilingApplication.getId());
+        sdkProfileDto.setApplicationToken(profilingApplication.getApplicationToken());
+        sdkProfileDto.setNotificationSchemaVersion(0);
+        sdkProfileDto.setLogSchemaVersion(0);
```

Comment:
Did you tried to build sandbox with this demo? You need to set notification, configuration and log schema versions to 1 (auto-generated ones) otherwise build will fail on SDK profile creation step.